src/styles/main.scss

```
@@ -1,3 +1,61 @@
 body {
   margin: 0;
 }
+
+$day-size: 100px;
+$month-lenght: 29;
+$start-days:
+  "mon" 1,
+  "tue" 2,
+  "wed" 3,
+  "thu" 4,
+  "fri" 5,
+  "sat" 6,
+  "sun" 7;
+
+.calendare {
+  position: relative;
+  width: 7*($day-size + 1);
+  display: flex;
+  flex-wrap: wrap;
+}
+
+@each $start-day, $num in $start-days {
+  .calendare--#{$start-day} .calendare__cell:first-child {
+    margin-left: ((101) * ($num - 1) + 1px);
+  }
+}
+
+.calendare__cell {
```

```suggestion
.&__cell {
```
use nesting everywhere